rtv/page.py

```diff
@@ -412,8 +412,23 @@ def edit(self):
             s.catch = False
             self.refresh_content()
 
+    @BaseController.register('i')
+    def get_inbox(self):
+        """
+        Checks the inbox for unread messages and displays a notification.
+        """
+        try:
+            if len(list(self.reddit.get_unread(limit=1))) > 0:
```

Comment:
I think you need to move the praw call outside of the if-else statement, otherwise it may get called twice.